kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -433,6 +433,34 @@ paths:
                 items:
                   $ref: '#/components/schemas/ConsumerGroup'
 
+    delete:
+      tags:
+        - Consumer Groups
+      summary: delete Consumer Groups
+      operationId: deleteConsumerGroups
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ConsumerGroupIds'
+
```

Comment:
Please remove empty line

Reply:
May be should we add endpoint: delete /api/clusters/{clusterName}/consumerGroups/{id} instead?